packages/core/include/combo/types.h

```diff
@@ -705,6 +705,9 @@ typedef enum {
 
 #define VTX(x,y,z,s,t,crnx,cgny,cbnz,a) { { { x, y, z }, 0, { s, t }, { crnx, cgny, cbnz, a } } }
 
+#define MASS_IMMOVABLE 0xFF // Cannot be pushed by OC colliders
```

Comment:
No C++ comment, no comment on the same line as the define as it can behave unexpectedly